sdk/node/src/lib.rs

```diff
@@ -114,15 +114,13 @@ impl<F: Farmer + 'static> Config<F> {
 
         let base = base.configuration(directory.as_ref(), chain_spec.clone()).await;
         let name = base.network.node_name.clone();
-        let database_source = base.database.clone();
 
-        let partial_components =
-            subspace_service::new_partial::<F::Table, RuntimeApi, ExecutorDispatch>(
-                &base,
-                &pot_external_entropy(&base, config_pot_external_entropy)
-                    .context("Failed to get proof of time external entropy")?,
-            )
-            .context("Failed to build a partial subspace node")?;
+        let partial_components = subspace_service::new_partial::<F::Table, RuntimeApi>(
+            &base,
+            &pot_external_entropy(&base, config_pot_external_entropy)
```

Comment:
I don't see `pot_external_entropy` function update in this PR, it was broken before and needs to be updated.